portality/static/js/doaj.fieldrender.edges.js

```diff
@@ -2195,9 +2195,15 @@ $.extend(true, doaj, {
                         this.currentQueryString = qs.queryString || "";
                     }
                 }
-                var frag = "<li class='alert'><p>You searched for <i>'";
+                var frag = "<li class='alert'><p>You searched for <i>";
```

Comment:
I'm in 2 minds, the apostrophes do add incorrect semantics to the search, looking like an exact phrase query, but it now looks a bit odd if you produce a search with only whitespace, e.g. a space.

Reply:
Ah I wasn’t sure why there was an extra typewriter apostrophe `'` — figured it was an error. Happy with either as long as the markup comes out nicely!

Reply:
OH Nevermind my comment — I understand now. Did not mean to remove it.

Reply:
Done in https://github.com/DOAJ/doaj/pull/2058/commits/19694a14b029024a6a6a9ec65ad71d307bff3bc5